Packages/com.chisel.editor/Chisel/Editor/Editor/Scene/CSGSceneBottomGUI.cs

```diff
@@ -13,670 +13,98 @@ namespace Chisel.Editors
 
-				rebuildContent = curSkin ? IconContent("icon_pro_rebuild") : IconContent("icon_pers_rebuild");
-				viewContent    = curSkin ? IconContent("icon_pro_grid")    : IconContent("icon_pers_grid"); // TODO: replace icon
+				toolbarStyle = new GUIStyle(EditorStyles.toolbar);
+				toolbarStyle.fixedHeight = kBottomBarHeight;
 
-				lockXButtonStyle = new GUIStyle(EditorStyles.toolbarButton);
-				lockXButtonStyle.normal  .textColor = Color.Lerp(UnityEditor.Handles.xAxisColor, curSkin ? Color.white : Color.black, 0.6f);
-				lockXButtonStyle.onNormal.textColor = Color.Lerp(UnityEditor.Handles.xAxisColor, curSkin ? Color.black : Color.white, curSkin ? 0.25f : 0.5f);
+				toggleStyle = new GUIStyle(EditorStyles.toolbarButton);
+				toggleStyle.fixedHeight = kBottomBarHeight;
 
-				lockYButtonStyle = new GUIStyle(EditorStyles.toolbarButton);
-				lockYButtonStyle.normal  .textColor = Color.Lerp(UnityEditor.Handles.yAxisColor, curSkin ? Color.white : Color.black, 0.6f);
-				lockYButtonStyle.onNormal.textColor = Color.Lerp(UnityEditor.Handles.yAxisColor, curSkin ? Color.black : Color.white, curSkin ? 0.25f : 0.5f);
+				buttonStyle = new GUIStyle(EditorStyles.toolbarButton);
+				buttonStyle.fixedHeight = kBottomBarHeight;
 
-				lockZButtonStyle = new GUIStyle(EditorStyles.toolbarButton);
-				lockZButtonStyle.normal  .textColor = Color.Lerp(UnityEditor.Handles.zAxisColor, curSkin ? Color.white : Color.black, 0.6f);
-				lockZButtonStyle.onNormal.textColor = Color.Lerp(UnityEditor.Handles.zAxisColor, curSkin ? Color.black : Color.white, curSkin ? 0.25f : 0.5f);
+				floatWidthLayout = GUILayout.Width(kFloatFieldWidth);
 
-				lockButtonToggleColor = new Color[2];
-				lockButtonToggleColor[0] = Color.white;
-				lockButtonToggleColor[1] = (curSkin) ? Color.Lerp(Color.white, Color.red, 0.5f) : Color.Lerp(Color.black, Color.red, 0.5f);
 				prevSkin = curSkin;
 				CSGEditorSettings.Load();
 			}
 
 
-			// Calculate size of bottom bar
+			// Calculate size of bottom bar and draw it
 			Rect position = sceneView.position;
 			position.x		= 0;
-			position.y		= position.height - (kBottomBarHeight + kTopBarHeight);
-			position.height = kBottomBarHeight;
-
-			try
-			{
-				UnityEditor.Handles.BeginGUI();
-				if (Event.current.type == EventType.Repaint)
-					EditorStyles.toolbar.Draw(position, false, false, false, false);
-
-				SceneViewBarState state;
-				if (!sceneViewStateLookup.TryGetValue(sceneView, out state))
-				{
-					state = new SceneViewBarState();
-					sceneViewStateLookup[sceneView] = state;
-				}
-
-				DrawBottomBar(position, state);
-
-				CSGEditorUtility.ConsumeUnusedMouseEvents(BottomBarGUIHash, position);
-			}
-			finally
-			{
-				UnityEditor.Handles.EndGUI();
-			}
-
+			position.y		= position.height - kBottomBarHeight;
+			position.height = kBottomBarHeight; 
 
+			GUILayout.Window(bottomBarGuiId, position, OnBottomBarUI, "", toolbarStyle);
```

Comment:
I forgot why I didn't use a Window .. but I remember having a good reason for it :-/